src/schema/basic_classes.yaml

```diff
@@ -375,12 +375,20 @@ classes:
       - was_informed_by
       - gold_analysis_project_identifiers
       - jgi_portal_analysis_project_identifiers
+      - replaces
+      - ended_at_time
     slot_usage:
       id:
         required: true
         structured_pattern:
           syntax: "{id_nmdc_prefix}:wfch-{id_shoulder}-{id_blade}$"
           interpolated: true
+      ended_at_time:
+        required: true
```

Comment:
Question about a general LinkML concept:

Is this slot an instance of this existing https://microbiomedata.github.io/berkeley-schema-fy24/ended_at_time/ slot? In other words, I can see here that it will be a required field, but will its value also have to match the regex pattern shown at https://microbiomedata.github.io/berkeley-schema-fy24/ended_at_time/?

Reply:
Good question.  Short answer is yes. That slot is defined as needing that regex in the slot definition.  Then on this particular class, we are further constraining the use of that slot by saying it is required (on this specific class, but not universally). It's probably worth adding an invalid test to demonstrate (and test!) this, I can do that.